scripts/local/examples/basic_send_receive.sh

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env bash
+# Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+# See the file LICENSE for licensing terms.
+
+set -e # Stop on first error
+
+# Variables provided by run_setup.sh:
+#   c_chain_rpc_url
+#   user_private_key
+#   user_address_bytes
+#   user_address
+#   relayer_address
+#   subnet_a_blockchain_id
+#   subnet_b_blockchain_id
+#   subnet_a_subnet_id
+#   subnet_b_subnet_id
+#   subnet_a_rpc_url
+#   subnet_b_rpc_url
+#   subnet_a_blockchain_id_hex
+#   subnet_b_blockchain_id_hex
+#   subnet_a_subnet_id_hex
+#   subnet_b_subnet_id_hex
+#   teleporter_contract_address
+#   warp_messenger_precompile_addr
+#   subnet_a_teleporter_registry_address
+#   subnet_b_teleporter_registry_address
+#   subnet_c_teleporter_registry_address
+
+# Test covers:
```

Comment:
what is the purpose for adding just the `basic_send_receive.sh` back vs the other previous test scripts?

Reply:
After working through a live demo recently, I realized that some reference commands for how to interact with Teleporter via bash would be extremely helpful. This script isn't intended to be run anywhere, nor is it intended to demonstrate anything more that very basic usage. Using this script as a starting point, a user could easily expand to deploying their own contracts, calling other Teleporter methods, etc.